### PR TITLE
[XcodeProject] Add llbuildSwift target in Xcode project

### PR DESCRIPTION
**Pull request by aciidb0mb3r**

### CODE REVIEW

File: llbuild.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2533,6 +2657,20 @@
 			shellPath = /bin/sh;
 			shellScript = "\"${SRCROOT}/utils/Xcode/create-target-link.sh\"";
 		};
+		BC8DEF2E20300F2000E9EF0C /* ShellScript */ = {
```
Comment (ddunbar) [inline llbuild.xcodeproj/project.pbxproj]: I think it would be nicer to use a copy files phase for this, if possible.

Comment (aciidb0mb3r) [reply llbuild.xcodeproj/project.pbxproj]: Didn't see an option for install dir in copy file phase :(

Comment (aciidb0mb3r) [reply llbuild.xcodeproj/project.pbxproj]: Actually, let me see if copy file phase expands macro in Absolute Path option.

Comment (aciidb0mb3r) [reply llbuild.xcodeproj/project.pbxproj]: Nah, can't use copy phase for files (.swiftmodule) inside BUILD_PRODUCT_DIR.

**Comment by aciidb0mb3r** [comment]
@swift-ci please smoke test

**Comment by aciidb0mb3r** [comment]
@swift-ci please smoke test

**Comment by aciidb0mb3r** [comment]
I am just going to merge this, the Xcode project has no effect on Swift's build process.

**Pull request merged**